src/models/mod.rs

```diff
@@ -0,0 +1,19 @@
+extern crate serde_json;
```

Comment:
I’m a but unsure of declaring this crate here instead of in the models using it. Errors would probably be raised if users import the model e.g User directly but not through this mod.

Reply:
In fact, since you already import those crates in `lib.rs`, they don't have to be imported again in `models/mod.rs`

Reply:
Whta do you prefer ?

Reply:
Let's keep it the way it is now

Reply:
Ok